frontend/src/components/utils/HeaderNavDropdown.jsx

```diff
@@ -0,0 +1,125 @@
+import { Dropdown } from "react-bootstrap";
+
+import IconAccounts from "../../../public/static/images/icon-accounts.svg";
+import IconBlocks from "../../../public/static/images/icon-blocks.svg";
+import IconTransactions from "../../../public/static/images/icon-transactions.svg";
+import IconNodes from "../../../public/static/images/icon-nodes.svg";
+
+const HeaderNavItem = ({ link, imgLink, text }) => {
+  return (
+    <Dropdown.Item className="header-nav-item" href={link}>
```

Comment:
We need to use the `<Link>` tag from next.js so the transitions are client-side instead of full reload.